gsw/gibbs/density_enthalpy_75.py

```diff
@@ -0,0 +1,194 @@
+v023 =  2.8695905159e-7
+v030 = -1.6521159259e-5
+v031 =  7.9279656173e-6
+v032 = -4.6132540037e-7
+v040 =  6.9111322702e-6
+v041 = -3.4102187482e-6
+v042 = -6.3352916514e-8
+v050 = -8.0539615540e-7
+v051 =  5.0736766814e-7
+v060 =  2.0543094268e-7
+v100 = -3.1038981976e-4
+v101 =  2.4262468747e-5
+v102 = -5.8484432984e-7
+v103 =  3.6310188515e-7
+v104 = -1.1147125423e-7
+v110 =  3.5009599764e-5
+v111 = -9.5677088156e-6
+v112 = -5.5699154557e-6
+v113 = -2.7295696237e-7
+v120 = -3.7435842344e-5
+v121 = -2.3678308361e-7
+v122 =  3.9137387080e-7
+v130 =  2.4141479483e-5
+v131 = -3.4558773655e-6
+v132 =  7.7618888092e-9
+v140 = -8.7595873154e-6
+v141 =  1.2956717783e-6
+v150 = -3.3052758900e-7
+v200 =  6.6928067038e-4
+v201 = -3.4792460974e-5
+v202 = -4.8122251597e-6
+v203 =  1.6746303780e-8
+v210 = -4.3592678561e-5
+v211 =  1.1100834765e-5
+v212 =  5.4620748834e-6
+v220 =  3.5907822760e-5
+v221 =  2.9283346295e-6
+v222 = -6.5731104067e-7
+v230 = -1.4353633048e-5
+v231 =  3.1655306078e-7
+v240 =  4.3703680598e-6
+v300 = -8.5047933937e-4
+v301 =  3.7470777305e-5
+v302 =  4.9263106998e-6
+v310 =  3.4532461828e-5
+v311 = -9.8447117844e-6
+v312 = -1.3544185627e-6
+v320 = -1.8698584187e-5
+v321 = -4.8826139200e-7
+v330 =  2.2863324556e-6
+v400 =  5.8086069943e-4
+v401 = -1.7322218612e-5
+v402 = -1.7811974727e-6
+v410 = -1.1959409788e-5
+v411 =  2.5909225260e-6
+v420 =  3.8595339244e-6
+v500 = -2.1092370507e-4
+v501 =  3.0927427253e-6
+v510 =  1.3864594581e-6
+v600 =  3.1932457305e-5
+
+
+@match_args_return
+def specvol(SA, CT, p):
+    """
+    Specific volume from SA, CT & p (75-term equation)
+
+    Calculates specific volume from Absolute Salinity, Conservative
+    Temperature and pressure, using the computationally-efficient 75-term
+    polynomial expression for specific volume (Roquet et al., 2015).
+
+    Parameters
+    ----------
+    SA : array_like
+         Absolute Salinity  [g/kg]
+    CT : array_like
+         Conservative Temperature [:math:`^\circ` C (ITS-90)]
+    p : array_like
+        sea pressure ( i.e. absolute pressure - 10.1325 dbar ) [dbar]
+
+    Returns
+    -------
+    specvol : array_like
+              in-situ density [m**3/kg]
+
+    Examples
+    --------
+    >>> import gsw
+    >>> SA = [34.7118, 34.8915, 35.0256, 34.8472, 34.7366, 34.7324]
+    >>> CT = [28.8099, 28.4392, 22.7861, 10.2261, 6.8272, 4.3235]
+    >>> p = [10, 50, 125, 250, 600, 1000]
+    >>> gsw.specvol(SA, CT, p)
+    array([ 0.00097863,  0.00097822,  0.00097616,  0.00097296,  0.00097103,
+            0.00096899])
+
+    Notes
+    -----
+    Note that this 75-term equation has been fitted in a restricted range of
+    parameter space, and is most accurate inside the "oceanographic funnel"
+    described in McDougall et al. (2003).  The GSW library function
+    "infunnel(SA,CT,p)" is avaialble to be used if one wants to test if some of
+    one's data lies outside this "funnel".
+
+    SA & CT need to have the same dimensions.
```

Comment:
It's not critical initially, but longer term we need to make the documentation a better match for the code: what we are actually doing is just letting the numpy broadcasting rules apply.  Therefore the statements about dimensions here are not quite right.  I would just delete everything about dimensions.